WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Notes Section/Add Order Note/NewNoteViewController.swift

```diff
@@ -105,9 +105,12 @@ private extension NewNoteViewController {
     private func loadSections() {
         let writeNoteSectionTitle = NSLocalizedString("WRITE NOTE", comment: "Add a note screen - Write Note section title")
         let writeNoteSection = Section(title: writeNoteSectionTitle, rows: [.writeNote])
-        let emailCustomerSection = Section(title: nil, rows: [.emailCustomer])
+        var emailCustomerSection: Section? = nil
+        if viewModel.order.billingAddress?.hasEmailAddress == true {
+            emailCustomerSection = Section(title: nil, rows: [.emailCustomer])
+        }
```

Comment:
So instead of using a variable, how about using a computed constant instead? Like so: 
```Swift
let emailCustomerSection: Section? = {
    if viewModel.order.billingAddress?.hasEmailAddress == true {
        return Section(title: nil, rows: [.emailCustomer])
    }
    return nil
}()
```
I think it would read better that way 🤔 WDYT?

Reply:
Definitely better! 🙌